introduction.md

```diff
@@ -1,3 +1,6 @@
 Hello!
 My favorite shizz thus far totally be the JavaScript path, yo!
 (even though I haven't actually done it yet) haha.
+* Git
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?